### PR TITLE
[Image] add css property

### LINKED ISSUE #598: [Image] add opacity property
We need a way to set `opacity` on Image component

### PR DESCRIPTION
**Pull request by alcferreira**
## Closes Issues

* Closes #598

### Reviewer 1

* [ ] Functions properly
* [ ] Good readability, clarity and overall quality
* [ ] Has tests
* [ ] Has docs

### Reviewer 2

* [ ] Functions properly
* [ ] Good readability, clarity and overall quality
* [ ] Has tests
* [ ] Has docs

### CODE REVIEW

**Comment by vNakamura** [comment]
Isn't this doing the same thing `style` already do?

**Comment by alcferreira** [comment]
Yep... I just want to avoid using the `style` tag

**Comment by vNakamura** [comment]
> Yep... I just want to avoid using the `style` tag

I don't get it. Any problem with `style` that I'm unaware of?

**Comment by alcferreira** [comment]
> > Yep... I just want to avoid using the `style` tag
> 
> I don't get it. Any problem with `style` that I'm unaware of?

Not necessary a problem, but it's a good practice to avoid using style
https://reactjs.org/docs/dom-elements.html#style

> Are inline styles bad?
CSS classes are generally better for performance than inline styles.

File: packages/image/src/Image.js [hunk 1]
```diff
@@ -20,7 +20,8 @@ Image.propTypes = {
   }),
   width: PropTypes.string,
   height: PropTypes.string,
-  fit: PropTypes.string
+  fit: PropTypes.string,
+  css: PropTypes.object
```
Comment (vNakamura) [inline packages/image/src/Image.js]: `styled-components` also works with `string` or with the ``css` `;`` helper (which returns an `array`).
Let's accept those as well.

```suggestion
  css: PropTypes.oneOfType([
    PropTypes.object,
    PropTypes.string,
    PropTypes.array
  ])
```

**Pull request merged**